scripts/o3de/o3de/manifest.py

```diff
@@ -14,8 +14,9 @@
 import os
 import pathlib
 import hashlib
+import urllib.parse
```

Comment:
It doesn't look like this import is actually used in this file and can be removed
```suggestion
```